### PR TITLE
PDA-179 - multi schema support

### PR DESCRIPTION
**Pull request by rimselis**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Comment by gReis89** [comment]
`ProcessDefinitionConfig.js` and `ProcessFormConfig.js` also uses UISchema, should we change it as well?

**Comment by rimselis** [comment]
> `ProcessDefinitionConfig.js` and `ProcessFormConfig.js` also uses UISchema, should we change it as well?

@gReis89 there is a separate task for that, as there seemed to be some issues with Process widget config loading that should be checked out as well

File: widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js]: can we use icons from [material-ui](https://material-ui.com/components/material-icons/)?

so we can get rid of that long svg's codes

File: widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js]: how can I create a new UI Schema from config screen?

Screen Shot 2020-05-05 at 09 50 22

**Comment by rimselis** [comment]
@gReis89 updated, unified the config connection/process fetching with PDA-240

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js:58]: can't we use the `this` keyword here instead of using the class name?

File: widgets/src/components/common/form/SchemaEditor/EditAddDialog.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/EditAddDialog.js]: we can use i18n here

File: widgets/src/components/common/form/SchemaEditor/SchemaContainer.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/SchemaContainer.js]: use i18n in this file as well

File: widgets/src/components/common/form/SchemaEditor/SchemaContainerControls.js
Comment (gReis89) [inline widgets/src/components/common/form/SchemaEditor/SchemaContainerControls.js]: i18n on titles and labels

File: widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js
Comment (rimselis) [reply widgets/src/components/common/form/SchemaEditor/JsonMultiFieldContainer.js:58]: As this function does not use component's `this` inside of it, it has to be a static function. Also, eslint would complain that class method is expected to use `this`.

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by rimselis** [comment]
@gReis89 I have translated the Completion Form config and same words used in other configs, left the rest of the words in other forms untouched (if there are any)

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Review by gReis89 — APPROVED** [review]
looks good!

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/115#pullrequestreview-406497192" title="Approved">gReis89</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
